src/lib/lib/collections/Collections.re

```diff
@@ -0,0 +1,292 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */;
+  /**
```

Comment:
Bad copy paste. Pasting a file after a ` */` in atom indents the entire file by one extra space :( it's so annoying.

Reply:
Good catch, thanks!